src/components/Skills/Skills.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './Skills.scss';
+import SkillCard from '../SkillCard/SkillCard';
+import img1 from '../../assets/pattern-graphic-design.svg'
+import img2 from '../../assets/pattern-illustrations.svg'
+import img3 from '../../assets/pattern-motion-graphics.svg'
+import img4 from '../../assets/pattern-photography.svg'
+import img5 from '../../assets/pattern-ui-ux.svg'
+import img6 from '../../assets/pattern-apps.svg'
+
+const Skills = () => {
+  return (
+    <div className="skills_grid">
+      <SkillCard image={img1} skill={"Graphic Design"}/>
```

Comment:
yes we can add the bgColor as a prop and setup utilities class for this.

Reply:
Not familiar with utility classes. Cant figure out how they work